site/docs/samples/notebook.md

```diff
@@ -1,162 +1,250 @@
+It demonstrates how policies are seamlessly applied when accessing the dataset. 
 
-## Before you begin
-
-Ensure that you have the following:
+In this sample you play multiple roles:
 
-- `kubectl` with access to a Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
-- Mesh for Data installed on your Kubernetes cluster
-- S3 Object storage account (e.g., Ceph, Minio, IBM Cloud Object Storage)
-- [Kubeflow](https://www.kubeflow.org/) installed on your cluster (this guide was tested with Kubeflow v1.0.2)
-- Web browser
+1. As a data ower you upload a dataset and register it in a data catalog
+2. As a data steward you setup data governance policies
+3. As a data user you specify your data usage requirements and use a notebook to consume the data
 
+## Before you begin
 
-!!! tip "Installing Kubeflow"
-    You can install Kubeflow on Kind by running [install_kubeflow.sh](https://github.com/ibm/the-mesh-for-data/blob/master/samples/kubeflow/install/kubeflow/install_kubeflow.sh). 
-    For OpenShift follow [KF_OPENSHIFT.md](https://github.com/ibm/the-mesh-for-data/blob/master/samples/kubeflow/install/kubeflow/KF_OPENSHIFT.md).
+- Install Mesh for Data using the [Quick Start](../get-started/quickstart.md) guide.
+  This sample assumes the use of the built-in catalog, Open Policy Agent (OPA) and flight module.
+- A web browser.
 
-## About this sample
-In this sample guide you will run a Kubeflow notebook with Mesh for Data and demonstrate that data read polices that are defined in Open Policy Agent (OPA) are seamlessly applied when reading a dataset.
+## Create a namespace for the sample
 
-In this sample guide you will:
+Create a new Kubernetes namespace and set it as the active namespace:
 
-1. Prepare a dataset to be accessed by the notebook
-1. Register the dataset in ODPi Egeria catalog
-1. Register the dataset credentials in Vault
-1. Deploy a Kubeflow notebook
-1. Create a Mesh for Data runtime environment for the notebook
-1. Read the dataset and observe policies applied seamlessly
+```bash
+kubectl create namespace m4d-notebook-sample
+kubectl config set-context --current --namespace=m4d-notebook-sample
+```
 
-## Getting started
+This enables easy [cleanup](#cleanup) once you're done experimenting with the sample.
 
-Completed all the steps in [quick start guide](../get-started/quickstart.md).
+## Prepare a dataset to be accessed by the notebook
 
-## Prepare the dataset for the sample notebook
+This sample uses the [Synthetic Financial Datasets For Fraud Detection](https://www.kaggle.com/ntnu-testimon/paysim1/data) dataset[^1] as the data that the notebook needs to read. Download and extract the file to your machine. You should now see a file named `PS_20174392719_1491204439457_log.csv`. Alternatively, you may download [`data.csv`](https://github.com/ibm/the-mesh-for-data/blob/master/samples/kubeflow/data.csv) which includes just the first 100 lines of this dataset.
 
```

Comment:
I suggest you add a footnote about "If you download your file from the location above, make sure you download the actual CSV data file and not an html view of the data.

Reply:
To which link you're referring to? data.csv from our repo? I will change it to match the original name of the asset in kaggle and will link to the raw file instead. Would that be sufficient?

Reply:
I was reffering to https://github.com/ibm/the-mesh-for-data/blob/master/samples/kubeflow/data.csv